packages/dependency-extraction-webpack-plugin/package.json

```diff
@@ -0,0 +1,33 @@
+{
+	"name": "@wordpress/dependency-extraction-webpack-plugin",
+	"version": "1.0.0",
+	"description": "Extract WordPress script dependencies from webpack bundles.",
+	"author": "The WordPress Contributors",
+	"license": "GPL-2.0-or-later",
+	"keywords": [
+		"wordpress",
+		"webpack",
+		"dependency"
+	],
+	"homepage": "https://github.com/WordPress/gutenberg/tree/master/packages/dependency-extraction-webpack-plugin/README.md",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/WordPress/gutenberg.git",
+		"directory": "packages/dependency-extraction-webpack-plugin"
+	},
+	"bugs": {
+		"url": "https://github.com/WordPress/gutenberg/issues"
+	},
+	"files": [
+		"index.js"
+	],
+	"main": "index.js",
+	"dependencies": {
+		"@wordpress/scripts": "file:../scripts",
+		"webpack": "4.8.3",
```

Comment:
`crypto` is not listed as a dependency

Reply:
The dependency is on Node `crypto`: https://nodejs.org/docs/latest-v10.x/api/crypto.html, not an npm package.